.spelling

```diff
@@ -166,6 +183,10 @@ searchfox
 SendPing
 SERP
 serverless
+signin
```

Comment:
Can we use the hyphenated versions of these words ('sign-in', 'sign-up', 'sign-ups') to avoid adding exceptions?